### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by kburnell**

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi**](https://that-website-git-timeline.that-conference.now.sh/wi) |
|:-:|
|<a href="https://that-website-git-timeline.that-conference.now.sh/wi">Screenshot of /wi</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/SL88UHo_Uns2KU1rOuxYevbJgsk_">(view full size)</a> |







Commit 297b7c713472082fbe8323dd5054302607d98ceb (https://that-website-git-timeline.that-conference.now.sh).

File: components/HomePage/Hero.js [hunk 1]
```diff
@@ -91,35 +91,15 @@ const SideDetail = styled.div`
...
+const Hero = ({ event, className }) => {
   return (
     <ContentSection className={className}>
       <Main>
         <SideDetail>
           <DateLocation>
             August 3 - 6, 2020 - Kalahari Resort, Wisconsin Dells, WI
           </DateLocation>
-          <Slogan>Summer Camp For Geeks</Slogan>
+          <Slogan>{event.slogan}</Slogan>
```
Comment (saragibby) [inline components/HomePage/Hero.js]: 👍

File: components/HomePage/Timeline.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import styled from 'styled-components';
+import moment from 'moment';
+import ContentSection from '../shared/ContentSection';
+import LinkButton from '../shared/LinkButton';
+import { below, DEFAULT_WIP_PAGE } from '../../utilities';
+
+const _ = require('lodash');
+
+const CalloutColor = '#05d69e';
```
Comment (saragibby) [inline components/HomePage/Timeline.js]: Be good to toss this into `baseTheme`.

File: components/HomePage/Timeline.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+`;
+
+const Date = styled(Detail)`
+  margin-top: -2rem;
+  width: 6rem;
+  margin-left: -1.5rem;
+  color: ${({ theme }) => theme.colors.fonts.light};
+
+  &.past {
+    color: #707070;
```
Comment (saragibby) [inline components/HomePage/Timeline.js]: You know what I'm thinking here right?? ...`baseTheme`

File: pages/wi/index.js [hunk 4]
```diff
@@ -1,47 +1,42 @@
...
 import SpeakerHighlight from '../../components/HomePage/SpeakerHighlight';
+import TimelineSection from '../../components/HomePage/Timeline';
 import SponsorHighlight from '../../components/HomePage/SponsorHighlight';
 import Testimonials from '../../components/HomePage/Testimonials';
-import ContentSection from '../../components/shared/ContentSection';
 import WhatToExpect from '../../components/shared/WhatToExpect';
 import NewsletterSignup from '../../components/HomePage/NewsletterSignup';
-import { below } from '../../utilities';
 
-const Title = styled.h2`
```
Comment (saragibby) [inline pages/wi/index.js]: Thanks for cleaning this up!

**Comment by kburnell** [comment]
@saragibby moved timeline colors to baseTheme

**Pull request merged**